lib/command/command-trait.php

```diff
@@ -24,6 +24,7 @@ trait Command_Trait {
 	/**
 	 * Gets the client transaction ID set for this command.
 	 *
+	 * @internal
```

Comment:
Maybe the trait itself could be `@internal`? When checking the docs, the trait doesn't have any properties, so I don't know if clients need to know about it

Edit: scratch that, I tested marking the trait as `@internal` and then its docs are just not generated, but the classes that reference it still show it